chia/daemon/client.py

```diff
@@ -79,10 +80,10 @@ async def timeout():
         asyncio.create_task(timeout())
         await self._request_dict[request_id].wait()
         if request_id in self.response_dict:
-            response = self.response_dict[request_id]
+            response: WsRpcMessage = self.response_dict[request_id]
             self.response_dict.pop(request_id)
         else:
-            response = None
+            response = None  # type: ignore
```

Comment:
i wanted to enable mypy for the rest of the file, however the logic already worked with this, so i left it as is.